config/initializers/airbrake.rb

```diff
@@ -1,7 +1,17 @@
-if !defined?(Rails) || Rails.env.staging? || Rails.env.production?
-  require 'airbrake'
+require 'airbrake'
 
+if !defined?(Rails) || Rails.env.staging? || Rails.env.production?
   Airbrake.configure do |config|
     config.api_key = ENV['AIRBRAKE_API_KEY']
   end
+else
+  module Airbrake
```

Comment:
Since this is just a development mode fix, keeping it simple is probably best. When an airbrake is thrown to the logs during development, the first thing I do is go directly to the line of code that threw it.